src/main/java/mate/academy/bookstore/validation/PasswordValidator.java

```diff
@@ -0,0 +1,20 @@
+package mate.academy.bookstore.validation;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+import java.util.Objects;
+
+public class PasswordValidator implements ConstraintValidator<FieldMatch, Object> {
+
+    @Override
+    public void initialize(FieldMatch constraintAnnotation) {
+        String firstObj = constraintAnnotation.first();
+        String secondObj = constraintAnnotation.second();
+    }
+
+    @Override
+    public boolean isValid(final Object firstObj,
+                           final ConstraintValidatorContext secondObj) {
+        return Objects.equals(firstObj, secondObj);
```

Comment:
Previous implementation was almost correct (revert)
```suggestion
        final Object firstObj = new BeanWrapperImpl(value).getPropertyValue(firstFieldName);
        final Object secondObj = new BeanWrapperImpl(value).getPropertyValue(secondFieldName);
        return Objects.equals(firstObj, secondObj);
```